ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,205 @@
+    await pressButton(page, 'Open as New Project');
+    const convertErrorMessage = await page
+      .getByTestId('info-modal-body')
+      .textContent();
+    const expectedErrorMessage =
+      'Convert error!\nGiven string could not be loaded as (query or plain) molecule or reaction, see the error messages: ' +
+      "'molecule auto loader: SMILES loader: cycle number 0 is not allowed', " +
+      "'scanner: BufferScanner::read() error', 'scanner: BufferScanner::read() error', " +
+      "'molecule auto loader: SMILES loader: cycle number 0 is not allowed'";
+    expect(convertErrorMessage).toEqual(expectedErrorMessage);
+  });
+
+  test('Open file - Open *.ket file', async ({ page }) => {
+    /*
+     * Test case: EPMLSOPKET-2933
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('KET/simple-chain.ket', page);
+  });
+
+  test('Open/Import structure as a KET file - create KET file', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-4702 (creating file)
+     */
+    const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+
+    await selectAtomInToolbar(AtomButton.Hydrogen, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const shiftForHydrogen = 150;
+    await dragMouseTo(x - shiftForHydrogen, y, page);
+    await resetCurrentTool(page);
+
+    await selectLeftPanelButton(LeftPanelButton.ReactionPlusTool, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const shiftForReactionPlus = 100;
+    await dragMouseTo(x - shiftForReactionPlus, y, page);
+    await clickInTheMiddleOfTheScreen(page);
+
+    await selectAtomInToolbar(AtomButton.Oxygen, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const shiftForOxygen = 40;
+    await dragMouseTo(x - shiftForOxygen, y, page);
+    await resetCurrentTool(page);
+
+    await selectLeftPanelButton(LeftPanelButton.ArrowOpenAngleTool, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    const shiftForCoordinatesToResetArrowOpenAngleTool = 100;
+    await page.mouse.move(x, y + shiftForCoordinatesToResetArrowOpenAngleTool);
+    await page.mouse.click;
+
+    await selectAtomInToolbar(AtomButton.Hydrogen, page);
+    const shiftForSecondHydrogen = 120;
+    await page.mouse.click(x + shiftForSecondHydrogen, y, {
+      button: 'left',
+    });
+  });
+
+  test('Open/Import structure as a KET file - open KET file', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-4702 (opening file)
+     * Test is related to the existing bug: https://github.com/epam/ketcher/issues/3153
+     */
+    await openFileAndAddToCanvas('KET/ket-4702-to-compare.ket', page);
```

Comment:
it is better to give the file a more understandable name: for example, 'two-atoms-benzene', so that there is an understanding of what kind of structure is used

Reply:
It's the reaction which I draw at previous test according to steps from jira. I don't know anything about chemistry so I can't tell what type of reaction it is :(